cmake/Modules/FindAubio.cmake

```diff
@@ -1,14 +1,15 @@
-
 if (NOT USE_SELF_BUILT_AUBIO)
     find_package(PkgConfig REQUIRED)
     if(PKG_CONFIG_FOUND)
         # The version we look for is legacy, may need some fine tunning
         pkg_check_modules(AUBIO IMPORTED_TARGET aubio>=0.4.9)
+            if(AUBIO_FOUND)
+                add_library(aubio ALIAS PkgConfig::AUBIO)
```

Comment:
This is causing an error for cmake < 3.19. I get the following error on Ubuntu 18.04 and 20.04, and on Fedora 32:
```
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.29.1") 
-- Checking for module 'aubio>=0.4.9'
--   Found aubio, version 0.4.9
CMake Error at cmake/Modules/FindAubio.cmake:7 (add_library):
  add_library cannot create ALIAS target "aubio" because target
  "PkgConfig::AUBIO" is imported but not globally visible.
  ```